features/user_should_be_able_to_create_articles.feature

```diff
@@ -4,26 +4,26 @@ Feature: User should be able to create articles
   I would like to be able to create an article on the site
 
   Scenario: User successfully creates an article
-    Given I am on the "Create Article" Page
+    Given I am on the "Create Article" page
     Then I fill in "Title" with "Holger is the best"
     And I fill in "Content" with "Dreamteam for the win!"
     Then I click on "Create Article"
     And I should see "Article successfully created"
 
   Scenario: User forgets to fill in content
-    Given I am on the "Create Article" Page
+    Given I am on the "Create Article" page
     Then I fill in "Title" with "Holger is the best"
     Then I click on "Create Article"
     And I should see "Body can't be blank"
 
   Scenario: User forgets to fill in Title
-    Given I am on the "Create Article" Page
+    Given I am on the "Create Article" page
     And I fill in "Content" with "Dreamteam for the win!"
     Then I click on "Create Article"
     And I should see "Title can't be blank"
 
 
   Scenario: User forgets to fill in Title
-    Given I am on the "Create Article" Page
+    Given I am on the "Create Article" page
```

Comment:
When writing the scenarios follow the Given, when, then convention.
If you need a reminder on it, go over this https://martinfowler.com/bliki/GivenWhenThen.html